### PR TITLE
use docker-compose in CI

### PR DESCRIPTION
**Pull request by smacker**
based on #117

### CODE REVIEW

File: scripts/start_docker_bblfsh.sh [hunk 1]
```diff
@@ -1,27 +0,0 @@
-#!/bin/bash
```
Comment (bzz) [inline scripts/start_docker_bblfsh.sh]: I find those 3 to be useful examples of how to run containers with all the options, etc individually with just `docker` that i.e may be useful for local debugging.

Do you think we could keep them around and delete, when this knowledge get's transferred to the  a Developer's docs? 

It's nice to have `docker-compose` to run all of them and I know that all the options, etc for individual docker command can be recovered from compose configuration, but I still see value for having those individual commands in some place.

Comment (smacker) [reply scripts/start_docker_bblfsh.sh]: don't see any value to keep scripts as documentation. but whatever.

**Comment by bzz** [comment]
Assume there still might be some WIP as CI seems to be red.

**Comment by smacker** [comment]
@bzz CI was red due to some internal travis problems. (tasks just hanged and died by timeout)
I have restarted it and everything is green again.

**Comment by smacker** [comment]
github didn't recognize that commits are in master already.

I have rebased just to be sure everything is ok.
Going to merge as CI pass again.

**Pull request merged**